src/home/buttons/index.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+import { CameraButton } from 'home/buttons/camera-button'
+import { DetectionButton } from 'home/buttons/detection-button'
+
+export const Buttons = () => <Grid container spacing={'10'} justify={'center'}>
```

Comment:
Would the names `Navigation` or `Live` be more appropriate here?

Reply:
done